composer/profiler/torch_profiler.py

```diff
@@ -54,84 +53,242 @@ class TorchProfiler(Callback):
+            +------------------------+-------------------------------------------------------+
+            | ``{world_size}``       | The world size, as returned by                        |
+            |                        | :func:`~.dist.get_world_size`.                        |
+            +------------------------+-------------------------------------------------------+
+            | ``{local_world_size}`` | The local world size, as returned by                  |
+            |                        | :func:`~.dist.get_local_world_size`.                  |
+            +------------------------+-------------------------------------------------------+
+            | ``{node_rank}``        | The node rank, as returned by                         |
+            |                        | :func:`~.dist.get_node_rank`.                         |
+            +------------------------+-------------------------------------------------------+
+            | ``{epoch}``            | The total epoch count, as returned by                 |
+            |                        | :meth:`~composer.core.time.Timer.epoch`.              |
+            +------------------------+-------------------------------------------------------+
+            | ``{batch}``            | The total batch count, as returned by                 |
+            |                        | :meth:`~composer.core.time.Timer.batch`.              |
+            +------------------------+-------------------------------------------------------+
+            | ``{batch_in_epoch}``   | The batch count in the current epoch, as returned by  |
+            |                        | :meth:`~composer.core.time.Timer.batch_in_epoch`.     |
+            +------------------------+-------------------------------------------------------+
+            | ``{sample}``           | The total sample count, as returned by                |
+            |                        | :meth:`~composer.core.time.Timer.sample`.             |
+            +------------------------+-------------------------------------------------------+
+            | ``{sample_in_epoch}``  | The sample count in the current epoch, as returned by |
+            |                        | :meth:`~composer.core.time.Timer.sample_in_epoch`.    |
+            +------------------------+-------------------------------------------------------+
+            | ``{token}``            | The total token count, as returned by                 |
+            |                        | :meth:`~composer.core.time.Timer.token`.              |
+            +------------------------+-------------------------------------------------------+
+            | ``{token_in_epoch}``   | The token count in the current epoch, as returned by  |
+            |                        | :meth:`~composer.core.time.Timer.token_in_epoch`.     |
+            +------------------------+-------------------------------------------------------+
```

Comment:
[nit] Could we use an f-string or substitution here to avoid copying this table from `composer.utils.file_helpers`?